hypertrace-gradle-java-convention-plugin/src/main/java/org/hypertrace/gradle/java/convention/JavaConventionExtension.java

```diff
@@ -8,6 +8,7 @@
 
 public class JavaConventionExtension {
   static final String EXTENSION_NAME = "javaConvention";
+  // TODO: resolve if need two different for source & target
   public final Property<JavaVersion> compatibilityVersion;
   public final Property<JavaLanguageVersion> toolchainVersion;
```

Comment:
Initially I used a enum (consisting of pair of these) to expose a single variable. But then moved to exposing two because if ever we want to configure compat to different than tool chain we need to have the flexibility and shouldn't require adding a new entry in enum

Reply:
Can we just use JavaVersion? It should directly translate to JavaLanguageVersion without us having to maintain a mapping/enum, no?